Input 컴포넌트 기능 추가, Sign Up 모달 수정

elppaaa opened the pull request:
- Input, Textarea 컴포넌트가 invalid, disabled 상태일 때, 스타일 추가
- 파이어폭스에서 fr단위를 읽지 못하여 %로 수정

<img width="261" alt="image" src="https://user-images.githubusercontent.com/15855011/94812458-badebc00-0431-11eb-8685-426431a756af.png">
<img width="295" alt="image" src="https://user-images.githubusercontent.com/15855011/94812467-bfa37000-0431-11eb-9d68-705a3989ea25.png">

elppaaa committed:
[fix] Correct path, Remove button outline

- ProfileModal 에서 경로 표시 오류 수정
- SigninButton outline 제거

fix #3

elppaaa committed:
[feat] Add faker package

elppaaa committed:
[feat] Add 'Input' styled, 'SignUpModal'

- 자주 쓰이는 Input styled 컴포넌트 추가
- 회원가입 Modal 추가 (View only)

develop #3

elppaaa committed:
[fix] Modify package.json, prevent Textarea resize

elppaaa committed:
[fix] Fixed grid error in Firefox

- 파이어폭스에서 fr단위를 읽지 못하여 %로 수정
- `ModalFrame`에서 인자로 받는 width를 최소 / 기본 너비로 수정

fix #3

elppaaa committed:
[feat] Add Input:invalid, Input:disalbed element

- Input, Textarea 컴포넌트가 invalid, disabled 상태일 때, 스타일 추가

elppaaa committed:
[feat] Add `UserInfoModal` (View)

develop #3

elppaaa commented:
__추가된 개인정보 화면입니다__
<img width="533" alt="image" src="https://user-images.githubusercontent.com/15855011/94928263-61908e80-04fe-11eb-811a-807205e1c8d1.png">
<img width="518" alt="image" src="https://user-images.githubusercontent.com/15855011/94928192-458ced00-04fe-11eb-8635-b396e33b4a17.png">

elppaaa committed:
[feat] Modify `UserInfoModal` View

- UserinfoModal 에서 editing 상태가 아닐 때에는 ValidCheck가 표시되지 않음

cyjo9603 commented on src/components/Input/Input.ts:
```diff
@@ -4,19 +4,41 @@ export const Input = styled.input`
   font-size: 0.7rem;
   padding: 0.1rem 0 0.1rem 0.3rem;
+
+  &:disabled {
+    background-color: ${({ theme }) => theme.DISABLED_BLUE};
+  }
+
+  &:invalid {
+    border: 1px ${({ theme }) => theme.ERROR} solid;
+    box-shadow: none;
+  }
```
이런 디테일 좋습니다 👍👍👍
`cursor: not-allowed;` 이런 커서도 있으면 ux가 좀 더 좋아질 것 같아요

cyjo9603 commented on src/components/Modals/UserinfoModal/UserinfoModal.tsx:
```diff
@@ -0,0 +1,72 @@
+  setEditing: (arg0: boolean) => void;
+}
+
+function Content({ isEditing, setEditing }: ContentProps): ReactElement {
```
컴포넌트를 선언할 땐 `function`이나 화살표 함수 중 아무거나 상관 없는데 스타일은 통일하는 게 좋습니다

elppaaa replied:
아 스니펫으로 생성했더니 알겠슴둥

cyjo9603 commented on src/components/Modals/UserinfoModal/UserinfoModal.tsx:
```diff
@@ -0,0 +1,72 @@
+        <span>대학 웹메일</span>
+        <span>
+          <Input disabled />
+          <span>@bu.ac.kr</span>
```
이메일 부분에 여백이 있으면 좋을것 같네요.
너무 바짝 붙어있는것 같아서...

elppaaa replied:
알겠슴둥

elppaaa replied:
✅

cyjo9603 reviewed (commented):
충돌난 코드들 모두 해결했고, 코드 리뷰 남겼습니다. 
모두 수정되면 그때 바로 스쿼시하겠습니다.

elppaaa committed:
[fix] change Input:disabled cursor

- `Input` 컴포넌트에서 disabled 일때, 커서 상태 변경
- SignUp, Userinfo Modal에서 email 간격 조정

fix #14

elppaaa committed:
[feat] Change `UserinfoModal` view

개인정보 확인 화면에서 패스워드, Input box가 불필요하다 판단하여 수정함

elppaaa commented:
두번째 feat 커밋에서 화면도 변경했어요 확인 부탁드려용 

<img width="533" alt="image" src="https://user-images.githubusercontent.com/15855011/94982655-5168c580-0577-11eb-8def-bdd2b443beab.png">

elppaaa committed:
[feat] Add `LoginModal` (View Only)

elppaaa committed:
[feat] Add `FindIdModal`, `FindPwModal`

elppaaa commented:
##### Modal 추가 했습니다.
- LoginModal
- FindIdModal
- FindPwModal

FindIdModal, FindPwModal은 같은 폴더 안에 넣으려고 했는데, 그래서 코드가 좀 전보다 난잡한 것 같기도 합니다. 어떤 스타일이 괜찮은지 모르겠네요. 피드백 부탁드립니다. 
디자인도 괜찮은지 모르겠네요 😢 
<img width="370" alt="image" src="https://user-images.githubusercontent.com/15855011/94987372-f77cf580-05a0-11eb-9208-5f5d4504ac8d.png">
<img width="418" alt="image" src="https://user-images.githubusercontent.com/15855011/94987375-ffd53080-05a0-11eb-8c9f-7e52fae54f28.png">
<img width="457" alt="image" src="https://user-images.githubusercontent.com/15855011/94987378-04014e00-05a1-11eb-907e-37aca81eee0c.png">
